beta-src/src/components/controllers/WDMainController.tsx

```diff
@@ -101,6 +103,14 @@ const WDMainController: React.FC = function ({ children }): React.ReactElement {
             // *not* yet seen (i.e. presumably they saw this phase when they were entering
             // orders for it, before that phase ended and other powers' orders appeared).
             dispatch(gameApiSliceActions.setViewedPhaseToLatestPhaseViewed());
+            if (status.status === "Left") {
```

Comment:
Unrelated to this change, but I think it's better to declare the function outside the JSX inside a `useCallback`, otherwise there may be rendering overhead. In the current form, a new clickhandler function is created every time this component renders.

Reply:
True, although I believe this component renders rarely (only on phase updae).